roxctl/help/help.properties

```
@@ -172,11 +172,12 @@ helm.output.long =
 helm.derive-local-values.short = Derive local Helm values from cluster configuration.
 helm.derive-local-values.long =
 
-generate.short = Commands related to generating different resources.
-generate.long =
+generate.short = (Technical Preview) Commands related to generating different resources.
+generate.long = Commands related to generating different resources.\n\n** This is a Technology Preview feature **\nTechnology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
```

There's a missing space. Also, the trailing dot becomes a part of URL, I think we should safely get rid of it.
```suggestion
generate.long = Commands related to generating different resources.\n\n** This is a Technology Preview feature **\nTechnology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/
```